content/developers/api-reference/public-scitt-api/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Public SCITT API (beta)"
+description: "Public SCITT API (beta) Reference"
+lead: "Public SCITT API (beta) Reference"
+date: 2023-11-07T11:56:23+01:00
+lastmod: 2023-11-07T11:56:23+01:00
+draft: false
+images: []
+menu: 
+  developers:
+    parent: "api-reference"
+weight: 112
+toc: true
+aliases: 
+  - /docs/api-reference/public-scitt-api/
+---
+{{< note >}}
+The public SCITT API is currently in beta.
+{{< /note >}}
+
+{{< note >}}
+This page is primarily intended for developers who will be writing applications that will use RKVST for provenance. 
+If you are looking for a simple way to test our API you might prefer our [Postman collection](https://www.postman.com/rkvst-official/workspace/rkvst-public-official/overview) or the [Developers](https://app.rkvst.io) section of the web UI. 
+
+{{< /note >}}
+## Public SCITT API Examples
+
+The Public SCITT API is based on the draft SCITT architecture: https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/main/draft-ietf-scitt-architecture.md
+
+While all RKVST events created by the Public SCITT API are public, in order to access the API an authorization token is needed.
+
+To create the authorization token follow these instructions: [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) and store in a file in a secure local directory with 0600 permissions.
+
+### Creating a Signed Statement
+
```

Comment:
I'm not the intended audience for this article but, having said that, I wouldn't know how to create a signed statement or what I would need to use to create one.

I'm pretty confident that I could follow the rest of the article but not this section.

Is it possible to have an examples of a statement json and how to wrap it so that the output matches the statement in 'Registering a Signed Statement'?

The quickest path to joy is to spoon feed an example and once they are hooked they can go an read the links.

Reply:
Thanks! I agree, have continued the discussion on slack :)